commit e2cc3a5e2afde6b0b97ecfcde08375a8550c554a
Author: Mark Boorer <markboo99@gmail.com>
Date:   Fri Jul 25 00:19:02 2014 +0100

    OCIOYaml: Fixed use of freed const char * on RHS of assignments - issue365

diff --git a/src/core/OCIOYaml.cpp b/src/core/OCIOYaml.cpp
index fc2f5f8..5a95353 100644
--- a/src/core/OCIOYaml.cpp
+++ b/src/core/OCIOYaml.cpp
@@ -1546,15 +1546,15 @@ OCIO_NAMESPACE_ENTER
                 {
                     std::vector<std::string> display;
                     load(second, display);
-                    const char* displays = JoinStringEnvStyle(display).c_str();
-                    c->setActiveDisplays(displays);
+                    std::string displays = JoinStringEnvStyle(display);
+                    c->setActiveDisplays(displays.c_str());
                 }
                 else if(key == "active_views")
                 {
                     std::vector<std::string> view;
                     load(second, view);
-                    const char* views = JoinStringEnvStyle(view).c_str();
-                    c->setActiveViews(views);
+                    std::string views = JoinStringEnvStyle(view);
+                    c->setActiveViews(views.c_str());
                 }
                 else if(key == "colorspaces")
                 {
